package.json

```diff
@@ -55,31 +48,34 @@
     "karma-mocha": "^1.0.1",
     "karma-mocha-reporter": "^2.0.3",
     "karma-webpack": "^1.7.0",
+    "mini-css-extract-plugin": "^1.6.0",
     "mocha": "^2.4.5",
-    "node-sass": "^4.6.0",
-    "postcss-loader": "^0.9.1",
+    "node-sass": "^6.0.0",
+    "postcss": "^8.2.15",
+    "postcss-loader": "^5.2.0",
     "react-addons-test-utils": "^15.0.2",
     "redux-devtools": "^3.3.1",
     "redux-devtools-dock-monitor": "^1.1.1",
     "redux-devtools-log-monitor": "^1.0.11",
     "request": "^2.74.0",
-    "sass-loader": "^3.2.0",
+    "sass-loader": "^11.1.0",
     "spritesh": "^1.2.0",
-    "style-loader": "^0.13.1",
-    "stylelint": "^5.4.0",
-    "stylelint-config-vtech": "^0.2.0",
-    "stylelint-webpack-plugin": "^0.2.0",
+    "style-loader": "^2.0.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-vtech": "^0.4.0",
+    "stylelint-webpack-plugin": "^2.1.1",
     "watch": "^0.18.0",
     "web-ext": "^1.10.0",
-    "webpack": "^1.13.0",
-    "webpack-dev-server": "^1.14.1",
-    "webpack-visualizer-plugin": "^0.1.5"
```

Comment:
I removed that since there is no proper version for webpack 5.
Also, it seems overkill to generate stats whenever a file change.